docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
+
+* Availability check
+
+== Requirements for availability check
+
+* PostgreSQL is accessible on network level
+* it's possible to login to PostgreSQL
+* it's possible to execute queries 
+
+== Solutions
+
+PostgreSQL managed by Stackgres:
+
+* it doesn't reuse generated connection string which our users will use, which could lead to situation where our monitoring is green, but users can't login (issues with Service)
+* it's sidecar, issue with exporter would require pod restarting
+* prometheusAutobind from SGCluster doesn't work, so I have a fear that whole monitoring isn't really well tested
```

Comment:
Have you tested with a ServiceMonitor? If it does not bind automatically maybe the ServiceMonitor resource is missing for some reason.

Reply:
I had installed prometheus-operator on the cluster, I set this option to True and there was no ServiceMonitor, no entry in logs that it was/wasn't create, no errors, nothing

Reply:
About wording: Be specific in decision documents. "fear" and "I" do not work in such a document. Be specific and well informed, no guessing is allowed here.